lib/MusicBrainz/Server/Entity/Editor.pm

```diff
@@ -104,6 +104,10 @@ sub is_editing_enabled {
     (shift->privileges & $EDITING_DISABLED_FLAG) == 0;
 }
 
+sub are_notes_disabled {
```

Comment:
This would mean that existing notes are disabled too but they are not.

```suggestion
sub is_adding_note_disabled {
```

Reply:
That sounds fine too though I'd keep `notes` plural